samples/snippets/feature_store_service/delete_featurestore_sample.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START aiplatform_delete_featurestore_sample]
+from google.cloud import aiplatform_v1beta1 as aiplatform
+
+
+def delete_featurestore_sample(
+    project: str,
+    featurestore_id: str,
+    location: str = "us-central1",
+    api_endpoint: str = "us-central1-aiplatform.googleapis.com",
+    timeout: int = 300,
+):
+    # The AI Platform services require regional API endpoints.
+    client_options = {"api_endpoint": api_endpoint}
+    # Initialize client that will be used to create and send requests.
+    # This client only needs to be created once, and can be reused for multiple requests.
+    client = aiplatform_v1beta1.FeaturestoreServiceClient(client_options=client_options)
```

Comment:
done

Reply:
Unsure why this should be `aiplatform.FeaturestoreServiceClient` - it's not available at the Vertex SDK or v1 layer, we should keep it as `aiplatform_v1beta1.FeaturestoreServiceClient` for clarity, using the import alias could be confusing to users who might try:
```py
from google.cloud import aiplatform

aiplatform.FeaturestoreServiceClient(...)  # Raises AttributeError
```

Reply:
This is because Morgan is working on the feature store SDK integration, and once done in the following week, we would simply remove the import alias without having to change the code. And it would be simpler for users to upgrade without changing their code.

Reply:
Got it - resolving this thread.